NetTopologySuite.IO.GeoPackage.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata>
+		<id>NetTopologySuite.IO.GeoPackage</id>
+		<version>$version$</version>
+		<title>NTS - Topology Suite - IO Library for GeoPackage</title>
+		<authors>NetTopologySuite - Team</authors>
+		<owners>NetTopologySuite - Team</owners>
+		<licenseUrl>https://github.com/NetTopologySuite/NetTopologySuite.IO.SpatiaLite/LICENSE.md</licenseUrl>
+		<projectUrl>https://github.com/NetTopologySuite/NetTopologySuite.IO.SpatiaLite</projectUrl>
+		<iconUrl>https://raw.githubusercontent.com/NetTopologySuite/GeoAPI/master/icon.png</iconUrl>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<description>
+      This package contains the GeoPackage IO library.
+    </description>
+		<summary>GeoPackage IO library for NetTopologySuite</summary>
+		<tags>NTS GeoPackage</tags>
+		<language>en-US</language>
+		<frameworkAssemblies>
+			<frameworkAssembly assemblyName="System" />
+			<frameworkAssembly assemblyName="System.Core" />
+		</frameworkAssemblies>
+		<dependencies>
+			<dependency id="GeoAPI" version="$geoapiversion$" />
```

Comment:
:exclamation:

This list of dependencies needs to match what we have in the .csproj.  Right now, it's lacking `NetTopologySuite`, but I have some other proposed changes that would affect this too.

See NetTopologySuite/NetTopologySuite#244 for an alternative, but that idea can wait until after the 2.x updates.

Reply:
ok, undestood